survey/SHACL_SurveyResultDataOffering.ttl

```diff
@@ -0,0 +1,52 @@
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix surveyonto: <http://semanticweb.org/metadatasurveyontology/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+
+surveyonto:InstantiatedVirtualResourceShape
+	a sh:NodeShape ; 
+	sh:targetClass surveyonto:InstantiatedVirtualResource ; 
+	sh:property [
+		sh:path surveyonto:survey_start_time ; 
+		sh:datatype xsd:dateTime ;  
+		skos:example "'Date 1'" ; 
+		sh:minCount 1 ; 
+		sh:message "Missing survey start time." ;
+		sh:name "Survey start time" ;
+		sh:description "When the survey was started." ;
+	] ;
+	sh:property [
+		sh:path surveyonto:survey_close_time ; 
+		sh:datatype xsd:dateTime ;  
+		skos:example "'Date 1'" ; 
+		sh:minCount 1 ; 
+		sh:message "Missing survey close time." ;
+		sh:name "Survey close time" ;
+		sh:description "When the survey was closed." ;
+	] ; 
+	sh:property [
+		sh:path surveyonto:survey_result_location ;
+		sh:datatype xsd:string ;
+		skos:example "'/mnt/data/survey_results/Result_xyz.json'" ;
+		sh:minCount 1 ;
+		sh:message "Missing survey result location." ;
+		sh:name "Survey result location" ;
+		sh:description "Location (server-side) where the survey result is saved." ;
+	] ;
+		sh:property [
+		sh:path surveyonto:contract_id ;
```

Comment:
as written above wie can leave this out if you nest the GeneralShape here (waiting for merge since review is not incorporated yet).